k8s/k8s.go

```diff
@@ -2069,6 +2073,27 @@ func (k *k8sOps) IsPodBeingManaged(pod v1.Pod) bool {
 	return false
 }
 
+func (k *k8sOps) ValidatePod(pod *v1.Pod) error {
+	t := func() (interface{}, bool, error) {
+		logrus.Infof("RK=> Pod Name: %s,\\n Pod Status: %s", pod.Name, pod.Status.Phase)
+		curr_pod, err := k.GetPodByName(pod.Name, pod.Namespace)
```

Comment:
Fetch the pod by its uuid instead of name. A different pod with the same name can show up. The users intent was to validate the exact pod they passed in.